index.php

```diff
@@ -307,7 +307,7 @@
                         <?php include('images/pantheon/actions/system-search.svg'); ?>
                         Applications
                     </div>
-                    <div id="slingshot-arrow"><img src='images/slingshot/arrow.svg'></div>
+                    <div id="slingshot-arrow"><img src='images/slingshot/arrow.svg' alt="An arrow that points to Applications name"></div>
```

Comment:
This doesn't really make sense to add an alt tag for since it's a part of a larger graphic, not a independent graphic in itself. It's essentially a border image

Reply:
Yes you're right! Decorative images shouldn't have any descripition, but it still needs and alt attribute, even if it is blank, so the code should be:
`<div id="slingshot-arrow"><img src='images/slingshot/arrow.svg' alt=""></div>`

Here are some info about using alt in decorative images [](https://webaim.org/techniques/alttext/#decorative)